loom-cli/main/loom.hs

```diff
@@ -217,7 +219,7 @@ buildLoom' logger buildConfig mode config home sitePrefix apx sf (BuildConfig ha
   firstT LoomSiteError $
     generateLoomSiteStatic sitePrefix siteRoot
   r <- firstT LoomError $
-    buildLoom (hoistLogger liftIO logger) buildConfig mode home (LoomTmp ".loom") config
+    buildLoom (hoistLogger liftIO logger) buildConfig mode home loomTmp config -- TODO: Extract
```

Comment:
TODO?

Reply:
Ah, that was accidentally added. (It was a `TODO` for the `LoomTmp` that got, well, extracted to `loomTmp`.)